src/spin1_beam_model/plotting.py

```diff
@@ -0,0 +1,144 @@
+import cmocean
+import healpy as hp
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+from mpl_toolkits.axes_grid1 import AxesGrid
+
+
+def StokesMatrix(n):
+    if n not in [0, 1, 2, 3]:
+        raise Exception("Input must be an integer in [0,1,2,3]")
+
+    if n == 0:
+        p = np.array([[1.0, 0], [0.0, 1.0]])
+    elif n == 1:
+        p = np.array([[1.0, 0], [0, -1.0]])
+    elif n == 2:
+        p = np.array([[0, 1.0], [1.0, 0]])
+    elif n == 3:
+        p = np.array([[0.0, -1j], [1j, 0]])
+
+    return p
+
+
+def MuellerMatrixElement(J, i, j):
+
+    Pi = StokesMatrix(i)
+    Pj = StokesMatrix(j)
+
+    M_ij = np.einsum("...ab,...bc,...cd,...ad", Pi, J, Pj, J.conj()) / 2.0
```

Comment:
`einsum` notation is always frustrating and opaque to me, but adding explanatory comments should probably be part of a larger documentation push for the repo. I'll make an issue for better docs.

Reply:
Yes, let's definitely make that issue.